target/linux/mvebu/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 BOARD:=mvebu
 BOARDNAME:=Marvell EBU Armada
 FEATURES:=fpu usb pci pcie gpio nand squashfs ramdisk boot-part rootfs-part
-SUBTARGETS:=cortexa9 cortexa53 cortexa72
+SUBTARGETS:=cortexa9 cortexa53 cortexa72 wdmc
```

Comment:
Why do you introduce this new subtarget?

Reply:
My reasoning was pretty much the same as in https://github.com/openwrt/openwrt/tree/master/target/linux/apm821xx with nand and sata subtargets: to isolate WD NAS-related stuff and Weltrend MCU related quirks. While there are at least 3 more devices like My Cloud Ultra I'd like work on and add under this subtarget I don't know if it's a good idea to group them into something generic like "sata" or "nas".

Reply:
I'm probably confused, because you're adding `wdmc` subtarget, but then you're still adding `Device/armada-385-wd-mcex2u` into `cortexa9` subtarget and into `wdmc` subtarget as well.

> My reasoning was pretty much the same as in https://github.com/openwrt/openwrt/tree/master/target/linux/apm821xx with nand and sata subtargets: 

You still didn't convinced me, that support for this device shouldn't go into `cortexa9` subtarget.

> to isolate WD NAS-related stuff and Weltrend MCU related quirks

I don't see any of these in this PR, can you be more specific?

Reply:
> I'm probably confused, because you're adding `wdmc` subtarget, but then you're still adding `Device/armada-385-wd-mcex2u` into `cortexa9` subtarget and into `wdmc` subtarget as well.

I'm sorry, I just missed it while I was working on moving it into subtarget.

> 
> > My reasoning was pretty much the same as in https://github.com/openwrt/openwrt/tree/master/target/linux/apm821xx with nand and sata subtargets:
> 
> You still didn't convinced me, that support for this device shouldn't go into `cortexa9` subtarget.
> 
> > to isolate WD NAS-related stuff and Weltrend MCU related quirks
> 
> I don't see any of these in this PR, can you be more specific?

Things like wtmcu package itself, wdmc/base-files/etc/diag.sh. And it might be more hairy in the future as other devices uses different MCU versions and indication logic.

Also I've used ARCH_WANT_LIBATA_LEDS, which does not do any good for devices without dedicated indicator for each drive. (It's still in mvebu though, in form of a patch as I don't understand how to isolate it into a subtarget properly. Need some advice here).

Reply:
> Things like wtmcu package itself

Isn't `DEVICE_PACKAGES += wtmcu` enough?

> wdmc/base-files/etc/diag.sh

I don't see a point of having separate subtarget just to use different diag.sh, you should probably extend current diag.sh in order to support your use case

> also I've used ARCH_WANT_LIBATA_LEDS, which does not do any good for devices without dedicated indicator for each drive.

Ideally this patch should be replaced with some upstreamable solution, but until then, I see no problem in adding this config option to `cortexa9` subtarget as I think, that kernel size and runtime overhead is negligible on this platform.

Reply:
Merged device into `cortexa9` subtarget.